packages/open-next/src/core/requestHandler.ts

```diff
@@ -65,6 +66,34 @@ export async function openNextHandler(
         delete headers[rawKey];
       }
 
+      if (
+        "isExternalRewrite" in preprocessResult &&
+        preprocessResult.isExternalRewrite
```

Comment:
```suggestion
        preprocessResult.isExternalRewrite === true
```

Reply:
Ho actually, this one cannot work, `preprocessResult` can be either an `InternalResult` or a `MiddlewareOutputEvent`